srml/council/src/seats.rs

```diff
@@ -164,7 +168,7 @@ decl_module! {
 			} else {
 				let imbalance = T::Currency::slash_reserved(&reporter, Self::voting_bond()).0;
 				T::BadReaper::on_unbalanced(imbalance);
-				Self::deposit_event(RawEvent::BadReaperSlashed(reporter));
```

Comment:
Incorrect. The actor here is called a *Reaper*, not a reporter.

In any case, please avoid code changes in PRs described as documentation only.

Reply:
Change reverted